src/common/src/types/mod.rs

```diff
@@ -143,7 +143,7 @@ pub enum DataType {
     #[from_str(regex = "(?i)^date$")]
     Date,
     #[display("character varying")]
-    #[from_str(regex = "(?i)^character varying$|^varchar$")]
+    #[from_str(regex = "(?i)^character varying$|^varchar$|^text$")]
```

Comment:
I thought this `from_str` was only used by unit tests. Could you point out how this is used by PG CDC?